tests/ResponseTest.php

```diff
@@ -259,4 +259,11 @@ public function testHeaderValuesAreTrimmed($r)
         $this->assertSame('Foo', $r->getHeaderLine('OWS'));
         $this->assertSame(['Foo'], $r->getHeader('OWS'));
     }
+
+    public function testCanConstructWithStatusCodeAndEmptyReason()
```

Comment:
This one looks good to me! I would maybe have preferred it at the top of the file together with `testCanConstructWithStatusCode` and `testCanConstructWithUndefinedStatusCode` but that is super nitpicky and more of a personal preference for grouping related tests.